src/Pages/Missions/Missions.jsx

```diff
@@ -0,0 +1,31 @@
+import { useSelector } from 'react-redux';
+import Mission from '../../Components/Mission/Mission';
+import './Missions.css';
+import './Missions.scss';
+
+const MissionPage = () => {
+  const missions = useSelector((state) => state.missions);
+
+  return (
+    <div className="missions-container">
+      <ul className="missions-titles">
+        <li>Mission</li>
+        <li>Description</li>
+        <li>Status</li>
+        <li />
+      </ul>
+      <ul className="missions">
+        {missions.map((mission) => (
+          <Mission
+            key={mission.div}
+            name={mission.name}
+            description={mission.description}
+            status="NOT A MEMBER"
+          />
+        ))}
```

Comment:
[Optional] You could add different background styles for each row to have a better UI.

![Screenshot_5](https://user-images.githubusercontent.com/88746054/208501613-e15c3325-a014-4180-9e79-438a9412da81.png)